From 55e03036ca78aa12e6ef2daa2f7b6184fd168e87 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 05 Jun 2022 18:30:23 +0300
Subject: [PATCH] Revert "[SW-77251] common: move round up macro to header"

Revert rom recovery implementation for secured firmware update via
inband and OOB.

This reverts commit 6fbcba24ff8668a671cf9ef16bd8cf8673753c2e.

Change-Id: I7c196215c7528e8da9bd03ad319dccbd6c11d86d
---

diff --git a/common/boot_info.c b/common/boot_info.c
index a9713a5..87bda09 100644
--- a/common/boot_info.c
+++ b/common/boot_info.c
@@ -14,6 +14,7 @@
 #include "boot_info.h"
 
 #define BOOT_INFO_BASE		(HL_GEN_MEM_BASE + HL_SCRATCHPAD_SRAM_OFFSET)
+#define ROUND_UP4(n)		(((n) + sizeof(uint32_t) - 1) & ~(sizeof(uint32_t) - 1))
 
 static struct boot_info boot_info = { .ptrs = { .count = 0 } };
 
diff --git a/include/arcboot.h b/include/arcboot.h
index 6156f9d..e9f1db2 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -32,7 +32,6 @@
 #define __unused	__attribute__((__unused__))
 #endif
 
-#define ROUND_UP4(n)			(((n) + sizeof(uint32_t) - 1) & ~(sizeof(uint32_t) - 1))
 #define BIT(nr)					(1 << (nr))
 #define ARRAY_SIZE(x)				(sizeof(x) / sizeof((x)[0]))
 /* extracts "n" number of bits from position "p" in the given number "v" */
